src/js/tabs/receiver.js

```diff
@@ -324,6 +324,56 @@ TABS.receiver.initialize = function (callback) {
             });
         });
 
+        // RC Smoothing
```

Comment:
I have no tested the code, but it seems strange to me not to have validation of MSP version to decide what actions execute, what elements show, etc.

Reply:
I do plan on adding that in as part of my cleanup, totally agree with you here